packages/plugin-ext/src/main/browser/command-registry-main.ts

```diff
@@ -72,7 +72,7 @@ export class CommandRegistryMainImpl implements CommandRegistryMain, Disposable
     }
 
     // tslint:disable-next-line:no-any
-    $executeCommand<T>(id: string, ...args: any[]): PromiseLike<T | undefined> {
+    $executeCommand<T>(id: string, args: any[]): PromiseLike<T | undefined> {
```

Comment:
@svenefftinge don't we need to align `CommandRegistryMain` interface?

Reply:
also, call sites at [1] looks suspicious. one is spreading, another isn't. 

[1] https://github.com/eclipse-theia/theia/blob/cff5bb5c4f915448af32500ade0ec3d3a1fa8a21/packages/plugin-ext/src/plugin/command-registry.ts#L108-L110

Reply:
@svenefftinge thanks for revising this!